dts/bindings/input/gpio-keys.yaml

```diff
@@ -16,3 +16,13 @@ child-binding:
        label:
           type: string
           description: Descriptive name of the key
+       zephyr,code:
+          type: int
+          default: 0
+          description: Key / Axis code to emit.
+       debounce-interval-ms:
```

Comment:
Within linux, there are some exceptions where the `debounce-interval` is different between gpio-keys.  Like in this example the power button has a higher debounce time, probably to prevent accidental shutdowns.
https://github.com/torvalds/linux/blob/7d0d3fa7339ed5a06d6608b7cde9f079eba62bb1/arch/arm/boot/dts/tegra30-cardhu.dtsi#L685

Reply:
You could technically partition the gpio keys into different sets that have different debounce constraints. Likely that would be a "lesser evil".